assets/src/scripts/contributors.js

```diff
@@ -267,4 +267,20 @@ jQuery( function ( $ ) {
 		updateRemoveButton();
 	};
 
+	const nameHeader = document.querySelector( 'th#name a' );
```

Comment:
Contributors table does not use `WP_List_Table` class. It is a Taxonomy.
We could replace the entire table using a custom rendering with `WP_List_Table` and use the `PressbooksTable` trait. I consider this solution taking into account that it is only to replace the `Name` sortable column header. The change was too small for a new rendering system. But I can consider this if the reviewer has other reasons I am not capturing.